meerk40t/gui/position.py

```diff
@@ -200,6 +211,10 @@ def update_position(self, reset):
             self.position_y = y0 / conversion
             self.position_w = (x1 - x0) / conversion
             self.position_h = (y1 - y0) / conversion
+            self.org_x = self.position_x
```

Comment:
The org_(x,y,w,h) parameters aren't defined in init. They are always set since it calls the code that sets them from init. But, some of the nanny code-checkers whine a little bit without their set to none originally code.